main/src/cgeo/geocaching/filters/core/NumberRangeGeocacheFilter.java

```diff
@@ -59,6 +60,22 @@ public void setMinMaxRange(final T min, final T max) {
         numberRangeFilter.setMinMaxRange(min, max);
     }
 
+    public <V extends Number & Comparable<V>> void setMinMaxRange(@NonNull final V leftValue, @NonNull final V rightValue,
+                                                                  @NonNull final V minValue, @NonNull final V maxValue,
+                                                                  @NonNull final Func1<V, T> valueConverter) {
+        boolean foundMinUnlimited = leftValue.compareTo(minValue) <= 0;
+        boolean foundMaxUnlimited = rightValue.compareTo(maxValue) >= 0;
+        if ((leftValue.equals(rightValue))
```

Comment:
I don't quite understand the meaning of this if statement. why would you set `foundmin/maxunlimited` to `false` under these circumstances, what is the intention for this? 
Also, this if statement would also set `foundmin/maxunlimited` to false if e.g. `leftValue` is equal to `rightValue`, but both are **outside** of the `minValue/maxvalue` range. Is this intended?

Reply:
If left value equals right value, there should be `value` displayed, and no comparison symbol. 

The outside situation occurs in the situation, when the value has a converting function (e.g. favorite percentage). If the value is entered via the dialog, left and right are not equal, but both >= max:
![grafik](https://user-images.githubusercontent.com/70113341/204026028-fa7e2594-ea86-40aa-a3c9-d1dcc04b9e9d.png)
In this case: without the enhanced if-statement `>= 100%` will be displayed - I thought `100%` will be better. Thats why this statement. Probably I will add some comments to it for better understanding

Reply:
Probably it depends will, on how have the filters are implemented, whether to display 100% or >= 100%
I will check that...

Reply:
If I understand correctly then you don't want to change anything about the filter's logic but just the display string in the topm is that correct?

Because in this case all your code should be done in method `getUserDisplayString`. The `setMinMaxValue` method, where you placed parts of your change, will also set the inner state of the filter, and thus your changes may also affect the filters logic (not only it's display string)

Reply:
Correct, I only wanted to change the display. 
Then I recognized the >= 100% issue and thought that could be enhanced.
But I wasn't really aware, that I changed the filter logic with my implementation  - so then we probably stay with the >= 100% and I will stay with changes only in `getUserDisplayString`

Reply:
I removed the code which changed the minMaxRange for the filter, but kept the new function `setMinMaxRange` for the logic when to set the limit to `null`

Reply:
LGTM code-wise, thanks @murggel.
The failing tests seem unrelated to me, should this be handled before a merge somehow?